charts/fybrik/notebook-test-readflow.tls.values.yaml

```diff
@@ -0,0 +1,81 @@
+# Default values for Fybrik.
+# This is a YAML-formatted file.
+# It overrides values that are different from the default values.yaml
+
+# Global configuration applies to multiple components installed by this chart
+global:
+  hub: localhost:5000/fybrik-system
+  tag: "0.0.0"
+  imagePullPolicy: "Always"
+
+  tls: 
+    enabled: true
+    mtlsEnabled: true
```

Comment:
mtls shuld be define per server, in the case of Fybrik per connector.

Reply:
OK, I will change that then.